packages/medusa/src/api/routes/admin/analytics-configs/create-analytics-config.ts

```diff
@@ -0,0 +1,32 @@
+import { IsBoolean } from "class-validator"
+import { Request, Response } from "express"
+import { EntityManager } from "typeorm"
+import { AnalyticsConfigService } from "../../../../services"
+import { CreateAnalyticsConfig } from "../../../../types/analytics-config"
+
+export default async (req: Request, res: Response) => {
```

Comment:
question: should we add a comment about the missing oas here so it's not accidentally added later on? (same for all endpoints)